src/modules/app/constants.js

```diff
@@ -1 +1,2 @@
-export const LANGUAGE_CODE_LENGTH = 2
+export const MIN_LANGUAGE_CODE_LENGTH = 2
```

Comment:
I think we could try to export whole enums: https://flow.org/en/docs/types/utilities/#toc-keys
Should we create a task for this?

Reply:
yes, I would appreciate it

Reply:
@maxammann did you create a task?